test/stringify.js

```diff
@@ -18,6 +18,11 @@ test('stringify()', function (t) {
         st.end();
     });
 
+    t.test('adds query prefix', function (st) {
+        st.equal(qs.stringify({ a: 'b' }, { addQueryPrefix: true }), '?a=b');
+        st.end();
+    });
```

Comment:
That seems like a reasonable modification - I haven't released the feature yet, so can you get a PR up asap for that?